site/labs/objects/objects.rst

```diff
@@ -2,194 +2,91 @@
 Objects
 *******
 
-Before Kattis
-=============
+* Feel free to use your laptop if you have it
+* Ensure I have recorded your completion --- failure to do so will result in a grade of 0
+* I strongly encourage you to work with others in the lab
 
-This lab relates to topics discussed in :doc:`class 13 </topics/objects/objects>` and :doc:`class 13b </topics/objects/collections>`.
+    * When you get stuck, do me a favour and ask those sitting around you for help
+    * I want people to get used to working together in the labs
+    * Peer teaching and peer learning is super effective
 
-Let's make a small program with classes. We'll make a program that will store a contact list of people. 
+.. note::
 
-We want 2 classes, and another script to interact with the objects we create. 
+    To obtain full marks for the lab, you must:
 
-Person
-======
+        #. Have completed the pre-lab exercises
+        #. Have been working on the lab content
+        #. Demonstrate competency in the topics
 
-Start by making a simple object to store information for a ``Person``. Create a ``Person`` class. 
 
-If using PyCharm or Spyder, open an empty python file in your working directory/project. Call it *Person.py*
+Pre Lab Exercises
+=================
 
-**Attributes**
+.. warning::
 
-* A first name (``_first_name``)
-* A last name (``_last_name``)
-* An email (``_email``)
+    You must have completed the specified exercises prior to the start of the lab. If you have not come to lab prepared,
+    you will be asked to leave and you will obtain a grade of 0 for the lab.
 
-There, that's it for now. We can add more later if we want. 
 
-**Methods**
+* Have a working implementation of the ``Circle`` class from :doc:`lecture </topics/objects/objects1>`
```

Comment:
They'd get a 0 if they don't copy/paste a file from the notes? Seems a little fussy.

Reply:
It's to ensure they have it ready to go as a template. I will add a note like _yes, it's a simple pre-lab exercise this week_.

Reply:
Still. To give 0 here seems excessive. It'll take like literally 10 seconds. You could even add a link to download the file.

Reply:
I feel ya. 
1. Removed warning about 0
2. Added download link